cmd/minishift/cmd/preflight_artifacts.go

```diff
@@ -59,10 +59,6 @@ func determineInitialClusterupParameters() []string {
 	clusterUpFlagSet.VisitAll(func(flag *flag.Flag) {
 		if viper.IsSet(flag.Name) {
 			key := flag.Name
-			_, exists := minishiftToClusterUp[key]
```

Comment:
where did `minishiftToClusterUp` come from?

Reply:
I wondered more about where it came from. Seems it is here: https://github.com/minishift/minishift/blob/2e48c0a652fba570829b842473dd9c42ddb48dbc/cmd/minishift/cmd/start.go#L121

so, we used it as global inside the package? yuck...

Reply:
@gbraad right, now it is removed.